rainier-core/src/main/scala/com/stripe/rainier/sampler/DualAvg.scala

```diff
@@ -7,7 +7,7 @@ final private class DualAvg(
     delta: Double,
     var logStepSize: Double,
     var logStepSizeBar: Double,
-    var avgAcceptanceProb: Double,
+    var avgError: Double,
```

Comment:
clarifying this value (Hbar in the paper): it's not the avg acceptance prob, it's the avg difference between the acceptance prob and delta.